assets/templates/homepage/main-figures-none.tmpl

```diff
@@ -0,0 +1,6 @@
+<div class="col col--lg-29 col--md-29">
```

Comment:
Padding/Margin on Main figures in the medium viewport looks a little off
![Screenshot 2020-10-21 at 14 06 10](https://user-images.githubusercontent.com/47502916/96723745-de4cb500-13a6-11eb-9432-9cf267596ad3.png)

Reply:
![image](https://user-images.githubusercontent.com/23668262/96737841-068fe000-13b6-11eb-9de5-8a77159dd788.png)

Should be fixed in latest commit, with a fix for the link too